### PR TITLE
ENG-1729: Updates to distinguish upstream and downstream content for DW

### PR DESCRIPTION
**Pull request by bredamc**
First set of updates, more to come

### CODE REVIEW

File: modules/configuring-distributed-workloads.adoc [hunk 1]
```diff
@@ -10,22 +10,36 @@ To configure the distributed workloads feature for your data scientists to use i
...
 * You have sufficient resources. In addition to the base {productname-short} resources, you need 1.1 vCPU and 1.6 GB memory to deploy the distributed workloads infrastructure.
 * You have access to a Ray cluster image. For information about how to create a Ray cluster, see the link:https://docs.ray.io/en/latest/cluster/getting-started.html[Ray Clusters documentation].
-* You have removed any previously installed instances of the CodeFlare Operator, as described in the Knowledgebase solution link:https://access.redhat.com/solutions/7043796[How to migrate from CodeFlare Operator to Red Hat OpenShift AI Cluster].
 ifndef::upstream[]
-* If you want to use graphics processing units (GPUs), you have enabled GPU support in {productname-short}. See link:{rhoaidocshome}{default-format-url}/managing_resources/managing-cluster-resources_cluster-mgmt#enabling-gpu-support_cluster-mgmt[Enabling GPU support in {productname-short}].
+* You have removed any previously installed instances of the CodeFlare Operator, as described in the Knowledgebase solution link:https://access.redhat.com/solutions/7043796[How to migrate from CodeFlare Operator to Red Hat OpenShift AI Cluster].
```
Comment (chtyler) [inline modules/configuring-distributed-workloads.adoc:14]: The KB article references RHOAI 2.4, but the steps to remove previously installed instances of the CodeFlare Operator should still be relevant. Anybody disagree?

Comment (bredamc) [reply modules/configuring-distributed-workloads.adoc:14]: I asked Dev whether to include that link in the ODH version or remove it, and was told to remove it.

File: modules/configuring-distributed-workloads.adoc [hunk 2]
```diff
@@ -10,22 +10,36 @@ To configure the distributed workloads feature for your data scientists to use i
...
+* If you want to use graphics processing units (GPUs), you have enabled GPU support in {productname-short}.
+See link:{rhoaidocshome}{default-format-url}/managing_resources/managing-cluster-resources_cluster-mgmt#enabling-gpu-support_cluster-mgmt[Enabling GPU support in {productname-short}].
```
Comment (chtyler) [inline modules/configuring-distributed-workloads.adoc:16]: As a matter of interest, do Habana Gaudi devices also support distributed workloads? If so, we'll need to get something added on Habana Gaudi devices in  the prereqs. Perhaps not in this PR, but in another PR at a future release.

File: modules/configuring-distributed-workloads.adoc [hunk 3]
```diff
@@ -10,22 +10,36 @@ To configure the distributed workloads feature for your data scientists to use i
...
+For more information, see https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/index.html[NVIDIA GPU Operator on {org-name} OpenShift Container Platform^] in the NVIDIA documentation.
 endif::[]
 
 .Procedure
 . In the {openshift-platform} console, click *Operators* -> *Installed Operators*.
+ifndef::upstream[]
 . Search for the *Red Hat OpenShift Data Science* Operator, and then click the Operator name to open the Operator details page.
+endif::[]
+ifdef::upstream[]
+. Search for the *Open Data Hub Operator*, and then click the Operator name to open the Operator details page.
```
Comment (chtyler) [inline modules/configuring-distributed-workloads.adoc:31]: Operator details should be bold as its a specific page?

*Operator details* page.

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc [hunk 4]
```diff
@@ -130,17 +141,33 @@ if __name__ == '__main__': <11>
 $ python compile_example.py
 ----
 This command creates a YAML file (in this example, `compiled-example.yaml`), which you can import in the next step.
+ifndef::upstream[]
 . Import your data science pipeline as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-data-science-pipelines_ds-pipelines#importing-a-data-science-pipeline_ds-pipelines[Importing a data science pipeline].
 . Schedule the pipeline run as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-data-science-pipelines_ds-pipelines#managing_pipeline_runs[Scheduling a pipeline run].
 . When the pipeline run is complete, confirm that it is included in the list of triggered pipeline runs as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-data-science-pipelines_ds-pipelines#viewing-triggered-pipeline-runs_ds-pipelines[Viewing triggered pipeline runs].
+endif::[]
+ifdef::upstream[]
+. Import your data science pipeline as described in link:{odhdocshome}/working-on-data-science-projects/#importing-a-data-science-pipeline_ds-pipelines[Importing a data science pipeline].
```
Comment (chtyler) [inline modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc:150]: Missing comma after pipeline?

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc [hunk 5]
```diff
@@ -130,17 +141,33 @@ if __name__ == '__main__': <11>
...
+. Schedule the pipeline run as described in link:{odhdocshome}/working-on-data-science-projects/#scheduling-a-pipeline-run_ds-pipelines[Scheduling a pipeline run].
```
Comment (chtyler) [inline modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc:151]: Missing comma after run?

File: modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc [hunk 6]
```diff
@@ -130,17 +141,33 @@ if __name__ == '__main__': <11>
...
 
 
 .Verification
 The YAML file is created and the pipeline run completes without errors.
+ifndef::upstream[]
 You can view the run details as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-data-science-pipelines_ds-pipelines#viewing-the-details-of-a-pipeline-run_ds-pipelines[Viewing the details of a pipeline run].
-
+endif::[]
+ifdef::upstream[]
+You can view the run details as described in link:{odhdocshome}/working-on-data-science-projects/#viewing-the-details-of-a-pipeline-run_ds-pipelines[Viewing the details of a pipeline run].
```
Comment (chtyler) [inline modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc:162]: Missing comma after details?

**Review by chtyler — APPROVED** [review]

File: modules/configuring-distributed-workloads.adoc (hunk 3 above)
Comment (bredamc) [reply modules/configuring-distributed-workloads.adoc:31]: The text "Operator details" is shown in a breadcrumb but not in the page title, so I thought it was better to use plain text (not bold) as a general description. Do you disagree?

Comment (chtyler) [reply modules/configuring-distributed-workloads.adoc:31]: No. Thanks for clarifying. It's fine.

File: modules/configuring-distributed-workloads.adoc (hunk 2 above)
Comment (bredamc) [reply modules/configuring-distributed-workloads.adoc:16]: This module is not the only module with that GPU prereq, so perhaps you should review all such modules in a separate PR, to see if an equivalent Habana Gaudi prereq should be added?

File: modules/configuring-distributed-workloads.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (bredamc) [reply modules/configuring-distributed-workloads.adoc]: Retitled to "How to migrate from a separately installed CodeFlare Operator in your data science cluster". 
Is that title ok with you?

[the remark this answers is not in the document]
Comment (chtyler) [reply modules/configuring-distributed-workloads.adoc]: That's fine thanks.

**Pull request merged**